#CREATE request parameters
string joint_name
--------------------------------
#CREATE response parameters
geometry_msgs/Twist cur_pos
